movieapp/src/containers/Header/Header.tsx

```diff
@@ -1,13 +1,62 @@
-import { SearchForm } from '../SearchForm/SearchForm';
-
-export function Header(): JSX.Element {
-  return (
-    <>
-      <h1>
-        <b>Netflix</b>Roulette
-      </h1>
-      <a href="#">Add Movie</a>
-      <SearchForm />
-    </>
-  );
+import {SearchForm} from '../SearchForm/SearchForm';
+import Modal from 'react-modal';
+import {useState} from "react";
+import {useTranslation} from "react-i18next";
+import {AddEditMovieForm} from "../AddEditMovieForm/AddEditMovieForm";
+import {Button} from "../../components/Button/Button";
+
+type HeaderProps = {
+    setMovies: React.MouseEventHandler<HTMLButtonElement>;
+}
+
+const customStyles = {
+    content: {
+        top: '50%',
+        left: '50%',
+        right: 'auto',
+        bottom: 'auto',
+        marginRight: '-50%',
+        transform: 'translate(-50%, -50%)',
+    },
+};
+
+export function Header(props:HeaderProps): JSX.Element {
+    const {t} = useTranslation();
+    const [modalIsOpen, setIsOpen] = useState(false);
+    const initialMovie = {title:'', description: '', key: ''};
```

Comment:
this constant will be recreated at each rerender.

it should not be here

Reply:
fixed